model/src/main/kotlin/Project.kt

```diff
@@ -155,7 +155,25 @@ data class Project(
     /**
      * A comparison function to sort projects by their identifier.
      */
-    override fun compareTo(other: Project) = id.compareTo(other.id)
+    override fun compareTo(other: Project): Int {
```

Comment:
Would it make sense to update the kDoc of the function to mention the definition file path?

Reply:
Absolutely.